config/routes.rb

```diff
@@ -62,9 +62,9 @@
   end
 
   # Funding Form pages
-  get "/funding-form" => "funding_form#index"
-  get "/funding-form/organisation-name" => "funding_form/organisation_name#show"
-  post "/funding-form/organisation-name" => "funing_form/organisation_name#submit"
+  get "/register-eu-funding-form" => "funding_form#index"
+  get "/register-eu-funding-form/organisation-name" => "funding_form/organisation_name#show"
+  post "/register-eu-funding-form/organisation-name" => "funing_form/organisation_name#submit"
```

Comment:
Not part of this PR, but:

```suggestion
  post "/register-eu-funding-form/organisation-name" => "funding_form/organisation_name#submit"
```